pom.xml

```diff
@@ -144,6 +144,11 @@
 			<artifactId>log4j</artifactId>
 			<version>1.2.8</version>
 		</dependency>
+		<dependency>
```

Comment:
[Nexus Lifecycle](https://www.sonatype.com/product-nexus-lifecycle) has found that this version of `bcprov-jdk15on` violates your company's policies.

Lifecycle recommends using version [1.60](https://search.maven.org/artifact/org.bouncycastle/bcprov-jdk15on/1.60/jar) instead as it does not violate any policies.